src/tests/SearchForm.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { fireEvent, getByLabelText, render, screen } from "@testing-library/react";
+import App from "../components/App";
+import SearchForm from "../components/SearchForm";
+import LocationDetails from "../components/LocationDetails";
+
+describe("Search Form", () => {
+  it("renders search button correctly", () => {
+    const validProps = {
+      searchText: {},
+      setSearchText: () => { },
+      onSubmit: () => { },
+    };
```

Comment:
You should move the valid props above the first `it` block, then you can re-use them for every `it` block inside this 
`describe` block. Also, if you look at what these props should be these could do with being re-written:
```suggestion
    const validProps = {
      searchText: "some Search Text",
      setSearchText: jest.fn(),
      onSubmit: jest.fn(),
    };
```

Reply:
amazing idea! makes it so much easier to read as well and means i am not having to enter data